util/downloadBuild.py

```diff
@@ -400,8 +403,9 @@ def defaultBuildType(repoName, arch, debug):
 
 def main():
     options = parseOptions()
+    if options.downloadFolder[-1] == '"': options.downloadFolder = options.downloadFolder[:-1];
```

Comment:
What does this do? (Comments would be nice :) )

Sidenote: This if block should be 2 lines for consistency, please.